layers/+lang/emacs-lisp/packages.el

```diff
@@ -160,6 +161,10 @@
       (spacemacs/set-leader-keys-for-major-mode 'emacs-lisp-mode
         "cl" 'auto-compile-display-log))))
 
+(defun emacs-lisp/init-elisp-def ()
+  (use-package elisp-def
+    :defer (spacemacs/defer)))
```

Comment:
normally we use :defer t
why do you use (spacemacs/defer) here?

Reply:
I don't know somehow other packages in the elisp layers follow this convention. Its goal I assume is to work with dump feature which I never try.